server/routerlicious/api-report/protocol-definitions.api.md

```diff
@@ -185,10 +185,6 @@ export interface IDocumentSystemMessage extends IDocumentMessage {
     data: string;
 }
 
-// @public (undocumented)
```

Comment:
@tylerbutler is this allowed. not sure what i'm supposed to do with these

Reply:
Yeah, this is expected since you're changing the public API. The idea is that those changes should be reviewed explicitly from an "API supportability" perspective.

Eventually we'll start failing PR builds if they change public APIs without also updating the API report file in the same PR - that's how we'll enforce reviews of public API changes.

There's more info [in the wiki](https://github.com/microsoft/FluidFramework/wiki/API-reports-and-review).